### PR TITLE
PR: Update Readme

### LINKED ISSUE #56: Update Readme
Now that we are very close to release 0.1, I think it's time to update our Readme with a simple description and a couple of screenshots.

@dalthviz, please take care of this one. You can take as inspiration:

https://github.com/spyder-ide/spyder-terminal/blob/master/README.md

but please leave the warning about this not being ready until we release 0.1. I'll remove that warning when we actually do the release.

### PR DESCRIPTION
**Pull request by dalthviz**
Fixes #56

### CODE REVIEW

**Comment by ccordoba12** [comment]
@dalthviz, please update the GIF to show the new *Recent notebooks* menu entry.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,47 @@
...
+To install this plugin, you can use either ``pip`` or ``conda`` package managers, as follows:
+
+Using pip:
+```
+pip install spyder-notebook
+```
+
+Using conda:
+```
+conda install spyder-notebook
```
Comment (goanpeca) [inline README.md]: `conda install spyder-notebook -c spyder-ide` ?

Comment (goanpeca) [inline README.md]: @ccordoba12 ?

**Comment by dalthviz** [comment]
@ccordoba12 
New gif:

notebook_example

File: README.md (hunk 1 above)
Comment (ccordoba12) [inline README.md]: Yes, I agree with both @goanpeca's recommendations. Please make the corresponding changes.

**Comment by ccordoba12** [comment]
I'll improve wording a bit before merging, but I think this is ready, thanks @dalthviz :-)

**Comment by ccordoba12** [comment]
@dalthviz, please update the animated gif to reflect the changes of the last couple of weeks, i.e.

1. The new switcher.
2. The menu action to open an IPython console connected to a notebook.

Then I'll merge this one.

**Comment by dalthviz** [comment]
Preview:

notebook_example

**Pull request merged**